bubble-wrap-style.yaml

```diff
@@ -4041,293 +4189,6 @@ layers:
                         visible: false
         #end different icon sprites
 
-        #most POI icons without name should be hidden, except these
```

Comment:
Ah is this tied to the `text-blend-order` removals above? (Or maybe I've lost track.)

Reply:
This was all moved server side, and the diff here makes that hard to read.